definitions/npm/express_v4.x.x/flow_>=v0.25.x/express_v4.x.x.js

```diff
@@ -12,7 +12,7 @@ declare module 'express' {
   }
   declare class Request extends http$IncomingMessage mixins RequestResponseBase {
     baseUrl: string;
-    body: mixed;
+    body: any;
```

Comment:
Yeah, this seems to be a problem, but I am wordering if we can add something like a type paramater 

```
declare class Request<B> extends http$IncomingMessage mixins RequestResponseBase {
  baseUrl: string;
  body: B;

}